src/main/kotlin/org/cascadebot/cascadebot/data/objects/GuildSettingsManagement.kt

```diff
@@ -15,9 +17,16 @@ class GuildSettingsManagement {
     @Setting
     var warnOver10Filters = true
 
+    
+
     val tags: ConcurrentHashMap<String, Tag> = ConcurrentHashMap()
     val filters: MutableList<CommandFilter> = mutableListOf()
     val permissions = GuildPermissions()
+    /*    get() {
+            assertWriteMode()
+            return field
+        }*/
+
```

Comment:
Need to look at why we did this

Reply:
I know why I did this, this was before we had the global write mode and permissions has a lot of writing so I just did it on everything, but then we added the global write mode and I added the assertWriteMode to the individual things so this is no longer needed.